### DISCUSSION TITLE
Minimize To Tray

### DISCUSSION THREAD

**Discussion by Tedfs3** [Rem0o/FanControl.Releases#747] [Q&A]
Is there an option to minimize the application back to the tray? I'm very new to FanControl so maybe I'm just not seeing the option somewhere.  If not, I'll change this to a feature request.

**Comment by Rem0o** [comment]
X out the window.

**Reply by Tedfs3** [reply]
For me, that closes the program.  Does not minimize to tray.

**Reply by Rem0o** [reply]
Are you sure? Look in task manager, FanControl.exe should still be active. Look carefully with your icons, maybe it's white on white so you don't see it?

**Reply by IslandJohn** [reply]
I'm having the same issue. It only minimizes to tray if I start FanControl with Windows (e.g. use the option).
